### DISCUSSION TITLE
Pause of scraping

### DISCUSSION THREAD

**Discussion by vukosim** [dsfsi/covid19za#837] [Q&A]
Morning @lrossouw wanted to check again in the scraping pause.

**Comment by lrossouw** [comment]
Sorry I think they change the format of the date in the URL:
They have been using (3 character months): 
`https://www.nicd.ac.za/latest-confirmed-cases-of-covid-19-in-south-africa-2-jun-2021/` 
But now use:
`https://www.nicd.ac.za/latest-confirmed-cases-of-covid-19-in-south-africa-2-june-2021/` 

I manually ran 1 June.  Will set it up to check that URL automatically going forward.

Also 2 June is a complete mess.  Will need to be captured manually:
* Cases have no province labels.
* Death/recovery table has been left out and captured as text.  Someone will have to work out new cumulative figures manually.

I see 3 June is back to normal.  If someone can capture 2 June then my script should pickup 3 June automatically.

**Reply by lrossouw** [reply]
Submitted PR.  Please review: #839

**Comment by vukosim** [comment]
Thanks, @lrossouw. I will also keep track. Just also having a life that is a bit upside down.
